### PR TITLE
Class based views

### PR DESCRIPTION
**Pull request by Jawayria**
Changed function based views to class based views and added Groups List View

### CODE REVIEW

File: cricPredict/Templates/create_group.html [hunk 1]
```diff
@@ -26,8 +26,22 @@ <h2>Create Group</h2>
                 </select>
               </div>
 
-               <input type="number" name="admin_id" id="admin_id" hidden value=1>
+               <input type="number" name="admin" id="admin" hidden value=1>
```
Comment (mraarif) [inline cricPredict/Templates/create_group.html]: I thought we removed this from the frontend

File: requirements.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@ djangorestframework==3.11.1
 idna==2.10
 optional-django==0.3.0
 pkg-resources==0.0.0
+psycopg2==2.8.5
```
Comment (mraarif) [inline requirements.txt:9]: did you manually add this here? or used `pip freeze`? (tip: you should always try to avoid changing requirements by hand)

**Review by mraarif — APPROVED** [review]
few questions otherwise it looks good!

File: cricPredict/Templates/create_group.html (hunk 1 above)
Comment (Jawayria) [reply cricPredict/Templates/create_group.html]: I forgot to push the updated code

File: requirements.txt (hunk 2 above)
Comment (Jawayria) [reply requirements.txt:9]: I used pip freeze

**Comment by mraarif** [comment]
go ahead and merge this, when ready

**Pull request merged**